### PR TITLE
Add a CI workflow that tests local builds.

### PR DESCRIPTION
**Pull request by pragmatrix**
Based on #527 this attempts to test local builds.

@therealbnut Can you take a look at the workflow file and the CI output. I think I am missing something. The binding generation step attempts to compile the bindings, but the ~~include directory seems to be missing~~ the include files can not be found.

### CODE REVIEW

**Comment by pragmatrix** [comment]
```bash
  cargo:rerun-if-env-changed=SKIA_DEBUG
  cargo:rerun-if-env-changed=SKIA_SOURCE_DIR
  cargo:rerun-if-env-changed=SKIA_LIBRARY_SEARCH_PATH
  STARTING BIND AGAINST SYSTEM SKIA
  cargo:rustc-link-search=tmp
  cargo:rerun-if-env-changed=SKIA_BUILD_DEFINES
  cargo:rerun-if-changed=src/bindings.cpp
  cargo:rerun-if-changed=src/svg.cpp
  cargo:rerun-if-changed=skia-bindings/skia/include
  cargo:rerun-if-env-changed=SDKROOT
  COMPILING BINDINGS: ["src/bindings.cpp", "src/svg.cpp"]
  running: "c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "skia-bindings/skia" "-Wall" "-Wextra" "-std=c++17" "-DSK_HAS_ANDROID_CODEC=" "-DNDEBUG=" "-DSK_R32_SHIFT=16" "-DSK_SUPPORT_GPU=0" "-DSK_GAMMA_APPLY_TO_A8=" "-DSKIA_IMPLEMENTATION=1" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_ENCODE_JPEG=" "-DSK_USE_LIBGIFCODEC=" "-DSK_CODEC_DECODES_PNG=" "-DSK_ENCODE_PNG=" "-DSK_XML=" "-o" "/home/armin/dev/rust-skia/target/release/build/skia-bindings-11fe44d305e7e816/out/skia/src/bindings.o" "-c" "src/bindings.cpp"
  cargo:warning=In file included from src/bindings.cpp:5:
  cargo:warning=src/bindings.h:4:10: fatal error: include/core/SkRefCnt.h: No such file or directory
  cargo:warning=    4 | #include "include/core/SkRefCnt.h"
  cargo:warning=      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
  cargo:warning=compilation terminated.
  exit code: 1
```

**Comment by therealbnut** [comment]
@pragmatrix I'm not 100% sure but perhaps if you could use an absolute path in the workflow file, my guess is that `skia-bindings` build is relative its `Cargo.toml`. You might be able to confirm this by emitting the working directory from rust.

File: Makefile [hunk 1]
```diff
@@ -91,3 +91,19 @@ github-build-win:
...
+#
+# https://github.com/rust-skia/rust-skia/pull/527
+
+.PHONY: test-local-build
+test-local-build:
+	cargo clean
+	cargo build --release
+	rm -rf tmp/
+	mkdir -p tmp/
+	find target -name "libskia*.a" -type f -exec cp {} tmp/ \;
```
Comment (therealbnut) [inline Makefile:107]: If this doesn't work try `target/debug/build/skia-bindings-*/out/skia/*.a` I'm not sure which specifically are needed but they might not all match your pattern, this is what's in my `out/skia`:

 * libfreetype2.a
 * libpng.a
 * libskparagraph.a
 * libbrotli.a
 * libharfbuzz.a
 * libskia-bindings.a
 * libskshaper.a
 * libexpat.a
 * libicu.a
 * libskia.a
 * libzlib.a

Comment (pragmatrix) [reply Makefile:111]: From what I know `libskia.a` will contain all the third party object files. With the feature `textlayout` enabled, `libskshaper.a` and `libskparagraph.a` are needed, but there is also a problem with a missing `icudtl.dat` that is expected to be in `OUT_DIR/skia`. I'll take a look at that later.

**Pull request merged**